docs/storage_snippets.py

```diff
@@ -138,7 +139,8 @@ def delete_blob(client, to_delete):
     from google.cloud.exceptions import NotFound
     client = storage.Client()
     bucket = client.get_bucket('my-bucket')
-    assert isinstance(bucket.list_blobs(), list)
+    blobs = list(bucket.list_blobs())
+    assert isinstance(blobs, list)
```

Comment:
yeah `my-bucket` has it's account locked. Or so the error tells me.

Also, other examples above could add blobs in the future so at this point I just assume you want more than 1.